README.md

```diff
@@ -20,6 +20,20 @@ export HOMEBREW_CORE_GIT_REMOTE="..."  # put your Git mirror of Homebrew/homebre
 
 The default Git remote will be used if the corresponding environment variable is unset.
 
+You can set `CI=true` to make the installation unattended. This will disable any interactive prompt, making it suitable for running on CI and on scripts. Note that some CI tools set this environment variable by default, [such as GitHub Actions](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables). Example:
+
+```bash
+CI=true /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
```

Comment:
@MikeMcQuaid I moved the PR to draft because I think I identified a flaw in my approach.

Regardless, could you please read the last paragraph I wrote in the README.md which contains the disclaimer about the HOMEBREW_PREFIX?

Also, both CI=true and CI=1 works fine. Actually, any value works as long as it's not empty, such as `CI=false`. I will document that instead.

Reply:
> Regardless, could you please read the last paragraph I wrote in the README.md which contains the disclaimer about the HOMEBREW_PREFIX?

Yeh, I did. I'm sorry but it's not sufficient to have this disclaimer but still have the behaviour work in the official installer non-interactively.

Reply:
I rephrased the CI sentence. Can you take a look again?

Reply:
But @MikeMcQuaid, many other users have faced this issue in the past, and I'm stating that it's not due to their decision. It's not that they prefer to install it to a different location, it's because there are no other choices.

I'm kindly: what are the caveats? How can we mitigate them? Should we raise a warning to the user at installation time when not using the default location? Should it be implemented as a cli as `--prefix` instead to make sure the user really knows what he's doing?